### PR TITLE
[TOPI] Minor fix in the LSTM recipe

### PR DESCRIPTION
**Pull request by junrushao1994**
This PR fixes the misbehavior of the LSTM recipe in Python 3, caused by `/` that should be `//`. By the way, it corrects the way to measure time cost, i.e. using `time_evaluator` instead of `time.time()`.

### CODE REVIEW

File: topi/recipe/rnn/lstm.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 """LSTM Example, still work in progress.."""
+from __future__ import division
```
Comment (vinx13) [inline topi/recipe/rnn/lstm.py]: you don't need to import this if you are not using `/` anymore

Comment (junrushao1994) [reply topi/recipe/rnn/lstm.py]: This is used some where
https://github.com/dmlc/tvm/pull/2131/files#diff-08e32b39e652ceee87d26c679fb8d747R46

Comment (vinx13) [reply topi/recipe/rnn/lstm.py]: in the link you posted that's floor div `//`, which is supported in Python2

Comment (junrushao1994) [reply topi/recipe/rnn/lstm.py]: I get what you mean...okay, i will remove the division thing.

**Comment by tqchen** [comment]
Thanks, @junrushao1994 @vinx13 @were! this is merged

**Pull request merged**